core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/JDBCUserStoreConstants.java

```diff
@@ -650,6 +653,24 @@ public class JDBCUserStoreConstants {
         setAdvancedProperty(JDBCRealmConstants.UPDATE_ROLE_NAME, "Update Group Name SQL",
                 JDBCRealmConstants.UPDATE_ROLE_NAME_SQL, "",
                 new Property[] { GROUP.getProperty(), SQL.getProperty(), FALSE.getProperty() });
+        setAdvancedProperty(JDBCRealmConstants.GET_GROUP_ID_FROM_GROUP_NAME, "Get Group ID From Group Name SQL",
+                JDBCRealmConstants.GET_GROUP_ID_FROM_GROUP_NAME_SQL, "",
+                new Property[] { GROUP.getProperty(), SQL.getProperty(), FALSE.getProperty() });
+        setAdvancedProperty(JDBCRealmConstants.GET_GROUP_NAME_FROM_GROUP_ID, "Get Group Name From Group ID SQL",
+                JDBCRealmConstants.GET_GROUP_NAME_FROM_GROUP_ID_SQL, "",
+                new Property[] { GROUP.getProperty(), SQL.getProperty(), FALSE.getProperty() });
+        setAdvancedProperty(JDBCRealmConstants.GET_GROUP_FROM_GROUP_NAME, "Get Group From Group Name SQL",
+                JDBCRealmConstants.GET_GROUP_FROM_GROUP_NAME_SQL, "",
+                new Property[] { GROUP.getProperty(), SQL.getProperty(), FALSE.getProperty() });
+        setAdvancedProperty(JDBCRealmConstants.GET_GROUP_FROM_GROUP_ID, "Get Group From Group ID SQL",
+                JDBCRealmConstants.GET_GROUP_FROM_GROUP_ID_SQL, "",
+                new Property[] { GROUP.getProperty(), SQL.getProperty(), FALSE.getProperty() });
+        setAdvancedProperty(JDBCRealmConstants.ADD_GROUP, "Add Group SQL",
+                JDBCRealmConstants.ADD_GROUP_SQL, "",
+                new Property[] { GROUP.getProperty(), SQL.getProperty(), FALSE.getProperty() });
+        setAdvancedProperty(JDBCRealmConstants.UPDATE_GROUP_NAME, "Update Group Name SQL",
+                JDBCRealmConstants.UPDATE_GROUP_NAME_SQL, "",
+                new Property[] { GROUP.getProperty(), SQL.getProperty(), FALSE.getProperty() });
```

Comment:
Don't these only need to be available in the unique id store?

Reply:
AFAIU having it here will inherit the config to non-uuid userstore managers

Reply:
There is no placeto maintain these properties in UniqueIDJDBCUserStore. Unique User ID related queries also kept in here.